### ISSUE TITLE
[Feature]: Compatible with tasks plug-in

### ISSUE THREAD

**Issue by lxmwwl** [dromse/obsidian-gamified-tasks#90]
### What kind of feature do you want?

I hope to be able to adapt to the repetitive task function of the tasks plug-in, and add a function to filter tasks based on the expiration time in the form of the tasks plug-in.

### Additional questions

- [ ] I want to work on this feature request

**Comment by dromse** [comment]
Hi, thanks for your suggestion! As I understood you want a feature to filter tasks by string-like `📅 2025-01-01` inside a task?

**Comment by lxmwwl** [comment]
> Hi, thanks for your suggestion! As I understood you want a feature to filter tasks by string-like `📅 2025-01-01` inside a task?您好，感谢您的建议！据我了解，您希望有一个功能可以在任务中按类似字符串`📅 2025-01-01`过滤任务？

使得，

**Comment by lxmwwl** [comment]
> Hi, thanks for your suggestion! As I understood you want a feature to filter tasks by string-like `📅 2025-01-01` inside a task?

Yes, want to be able to sort tasks based on this and loop tasks based on 🔁

**Comment by dromse** [comment]
Okay, I'll try to add this feature in the future releases. 😄
